tables/es-us-g1.ctb

```diff
@@ -0,0 +1,31 @@
+#
+#  Copyright (C) 2017 American Printing House for the Blind
+#
+#  This file is part of liblouis.
+#
+#  liblouis is free software: you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as
+#  published by the Free Software Foundation, either version 2.1 of the
+#  License, or (at your option) any later version.
+#
+#  liblouis is distributed in the hope that it will be useful, but
+#  WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with liblouis. If not, see
+#  <http://www.gnu.org/licenses/>.
+#
+# Table for Spanish Braille used in the US.
+#
+# The idea with this table is to define the differences from standard Spanish Braille.
+# It is unclear whether there really is a formal standard for Spanish US and so this is based upon user feedback.

```

Comment:
as far as I am aware, there is none. Though there certainly is in Latin America (most sources are listed on pr 653) Though it would be good either to adhere to one (as most texts are grade one anyway) to make language learning as straightforward as possible.